### ISSUE TITLE
Onscreen keyboard reports error "cannot launch xvkbd"

### ISSUE THREAD

**Issue by igor2x** [flathub/org.pwsafe.pwsafe#6]
Installed Password Safe v1.18.2 as flatpak from Flathub on Ubuntu 23.10.

Start Password Safe and click on keyboard icon next to the Safe Combination input box and error is displayed: 
 "Could not launch external onscreen keyboard. Cound not launch xvkbd. Please make sure it's installed and in your PATH."

Probably the cause of the problem is because when using flatpak, any package like xvkbd should bi installed inside flatpak package itself. Maybe for start change the message to express, this is not supported in flatpak at the moment.

**Comment by ronys** [comment]
I'd rather not make a flatpak-specific error message. Easier to add xvkbd to the manifest.

**Comment by igor2x** [comment]
I agree, if it is simpler to add xvkbd to manifest, then to need to add some specific error.

I have searched the web and it looks like this is the xvkbd project web site: http://t-sato.in.coocan.jp/xvkbd/
At the bottom of screen is "Download" section where it looks it is access to source code when we decide to add it to flatpak.

But I don't know if it is worth the effort before we are sure this xvkbd works fine with Password Safe using deb package. I have reported a bug on [Password Safe bug tracker](https://github.com/pwsafe/pwsafe/issues/1103), because I can't make onscreen keyboard to work at all on Password Safe deb version of a program. I imagine if it is not working on deb, then we will probably have even more issues working it using flatpak.
